tsconfig.base.json

```diff
@@ -17,6 +17,7 @@
     "noUnusedLocals": true,
     "forceConsistentCasingInFileNames": true,
     "lib": ["es2019", "esnext.asynciterable"],
+    "skipLibCheck": true,
```

Comment:
It's not clear what the motivation for this is or if this could be applied more granularly, e.g., at a specific package's own `tsconfig.json` which inherit from this base config. 

I'm curious what the motivation is, but also just wanted to point out an often-unknown detail that `tsconfig.json` files are parsed with a [jsonc parser](https://komkom.github.io/) which supports comments.  Basically, I think a comment here noting the motivation for that would survive in this file would be really beneficial. (e.g., is it for performance, to allow the application to compile, etc?)